tests/js/utils.js

```diff
@@ -378,17 +378,16 @@ export const provideModuleRegistrations = ( registry, extraData = [] ) => {
  * @since 1.42.0
  *
  * @param {Object}  registry             Registry object to dispatch to.
- * @param {Object}  data                 List of module registration data objects to be merged with defaults. Default empty array.
+ * @param {Object}  survey               The current survey.
  * @param {Object}  args                 Optional arguments.
  * @param {boolean} args.trackingEnabled Optional. Whether the tracking should be enabled or not.
- * @param {string}  args.triggerID       Optional. Survey trigger ID.
  */
 export function provideCurrentSurvey(
 	registry,
-	data,
-	{ triggerID = 'testSurvey', trackingEnabled = true } = {}
+	survey,
```

Comment:
Let's provide a default `survey` definition here while allowing its fields to be customized by the caller like the other `provide*` helpers. I think we could even combine `survey` and the `options` into a single second argument to achieve that and simply pull out the non-survey value(s) from the received/final survey object. Does that make sense?

Reply:
Reworked the provideCurrentSurvey function. Removed tracking part from it because it doesn't make sense to be there and added the default value for the `survey` prarameter.